Cannot build on OSX Maverics

When building on OSX 10.9, I get the following errors on make

```
src/xmpp/capabilities.c:211:5: error: 'OpenSSL_add_all_digests' is deprecated: first deprecated in OS X 10.7
      [-Werror,-Wdeprecated-declarations]
    OpenSSL_add_all_digests();
    ^
/usr/include/openssl/evp.h:840:6: note: 'OpenSSL_add_all_digests' declared here
void OpenSSL_add_all_digests(void) DEPRECATED_IN_MAC_OS_X_VERSION_10_7_AND_LATER;
     ^
src/xmpp/capabilities.c:212:10: error: 'EVP_get_digestbyname' is deprecated: first deprecated in OS X 10.7
      [-Werror,-Wdeprecated-declarations]
    md = EVP_get_digestbyname("SHA1");
         ^
/usr/include/openssl/evp.h:849:15: note: 'EVP_get_digestbyname' declared here
const EVP_MD *EVP_get_digestbyname(const char *name) DEPRECATED_IN_MAC_OS_X_VERSION_10_7_AND_LATER;
              ^
src/xmpp/capabilities.c:213:5: error: 'EVP_MD_CTX_init' is deprecated: first deprecated in OS X 10.7
      [-Werror,-Wdeprecated-declarations]
    EVP_MD_CTX_init(&mdctx);
    ^
/usr/include/openssl/evp.h:547:6: note: 'EVP_MD_CTX_init' declared here
void    EVP_MD_CTX_init(EVP_MD_CTX *ctx) DEPRECATED_IN_MAC_OS_X_VERSION_10_7_AND_LATER;
        ^
src/xmpp/capabilities.c:214:5: error: 'EVP_DigestInit_ex' is deprecated: first deprecated in OS X 10.7
      [-Werror,-Wdeprecated-declarations]
    EVP_DigestInit_ex(&mdctx, md, NULL);
    ^
/usr/include/openssl/evp.h:555:5: note: 'EVP_DigestInit_ex' declared here
int     EVP_DigestInit_ex(EVP_MD_CTX *ctx, const EVP_MD *type, ENGINE *impl) DEPRECATED_IN_MAC_OS_X_VERSION_10_7...
        ^
src/xmpp/capabilities.c:215:5: error: 'EVP_DigestUpdate' is deprecated: first deprecated in OS X 10.7
      [-Werror,-Wdeprecated-declarations]
    EVP_DigestUpdate(&mdctx, s->str, strlen(s->str));
    ^
/usr/include/openssl/evp.h:556:5: note: 'EVP_DigestUpdate' declared here
int     EVP_DigestUpdate(EVP_MD_CTX *ctx,const void *d,
        ^
src/xmpp/capabilities.c:216:5: error: 'EVP_DigestFinal_ex' is deprecated: first deprecated in OS X 10.7
      [-Werror,-Wdeprecated-declarations]
    EVP_DigestFinal_ex(&mdctx, md_value, &md_len);
    ^
/usr/include/openssl/evp.h:558:5: note: 'EVP_DigestFinal_ex' declared here
int     EVP_DigestFinal_ex(EVP_MD_CTX *ctx,unsigned char *md,unsigned int *s) DEPRECATED_IN_MAC_OS_X_VERSION_10_...
        ^
src/xmpp/capabilities.c:217:5: error: 'EVP_MD_CTX_cleanup' is deprecated: first deprecated in OS X 10.7
      [-Werror,-Wdeprecated-declarations]
    EVP_MD_CTX_cleanup(&mdctx);
    ^
/usr/include/openssl/evp.h:548:5: note: 'EVP_MD_CTX_cleanup' declared here
int     EVP_MD_CTX_cleanup(EVP_MD_CTX *ctx) DEPRECATED_IN_MAC_OS_X_VERSION_10_7_AND_LATER;
        ^
7 errors generated.
make: *** [src/xmpp/capabilities.o] Error 1
```

--- a/configure.ac
+++ b/configure.ac
@@ -60,8 +60,9 @@ AM_CPPFLAGS="$AM_CPPFLAGS $PARSER_CFLAGS"
 LIBS="$LIBS $PARSER_LIBS"
 
 ### Check for libstrophe dependencies
-AC_CHECK_LIB([resolv], [__res_query], [],
-    [AC_MSG_ERROR([libresolv is required for profanity])])
+AC_CHECK_LIB([resolv], [res_query], [],
+    [AC_CHECK_LIB([resolv], [__res_query], [],
+        [AC_MSG_ERROR([libresolv is required for profanity])])])
 PKG_CHECK_MODULES([openssl], [openssl], [],
     [AC_MSG_ERROR([openssl is required for profanity])])
 AM_CPPFLAGS="$AM_CPPFLAGS $openssl_CFLAGS"

